include/openssl/err.h

```diff
@@ -108,49 +108,48 @@ typedef struct err_state_st {
+#  define OCSPerr(f,r) ERR_raise(ERR_LIB_OCSP,(r))
+#  define OSSL_STOREerr(f,r) ERR_raise(ERR_LIB_OSSL_STORE,(r))
+#  define PEMerr(f,r) ERR_raise(ERR_LIB_PEM,(r))
+#  define PKCS12err(f,r) ERR_raise(ERR_LIB_PKCS12,(r))
+#  define PKCS7err(f,r) ERR_raise(ERR_LIB_PKCS7,(r))
+#  define PROPerr(f,r) ERR_raise(ERR_LIB_PROP,(r))
+#  define PROVerr(f,r) ERR_raise(ERR_LIB_PROV,(r))
+#  define RANDerr(f,r) ERR_raise(ERR_LIB_RAND,(r))
+#  define RSAerr(f,r) ERR_raise(ERR_LIB_RSA,(r))
+#  define SM2err(f,r) ERR_raise(ERR_LIB_SM2,(r))
+#  define SSLerr(f,r) ERR_raise(ERR_LIB_SSL,(r))
+#  define SYSerr(f,r) ERR_raise(ERR_LIB_SYS,(r))
+#  define TSerr(f,r) ERR_raise(ERR_LIB_TS,(r))
+#  define UIerr(f,r) ERR_raise(ERR_LIB_UI,(r))
+#  define X509V3err(f,r) ERR_raise(ERR_LIB_X509V3,(r))
+#  define X509err(f,r) ERR_raise(ERR_LIB_X509,(r))
 # endif
-# define FUNCerr(f,r)  ERR_raise_data(ERR_LIB_SYS,(r),"calling function %s",(f))
-# define BNerr(f,r)   ERR_raise(ERR_LIB_RSA,(r))
-# define RSAerr(f,r)  ERR_raise(ERR_LIB_RSA,(r))
-# define DHerr(f,r)   ERR_raise(ERR_LIB_DH,(r))
-# define EVPerr(f,r)  ERR_raise(ERR_LIB_EVP,(r))
-# define BUFerr(f,r)  ERR_raise(ERR_LIB_BUF,(r))
-# define OBJerr(f,r)  ERR_raise(ERR_LIB_OBJ,(r))
-# define PEMerr(f,r)  ERR_raise(ERR_LIB_PEM,(r))
-# define DSAerr(f,r)  ERR_raise(ERR_LIB_DSA,(r))
-# define X509err(f,r) ERR_raise(ERR_LIB_X509,(r))
-# define ASN1err(f,r) ERR_raise(ERR_LIB_ASN1,(r))
-# define CONFerr(f,r) ERR_raise(ERR_LIB_CONF,(r))
-# define CRYPTOerr(f,r) ERR_raise(ERR_LIB_CRYPTO,(r))
-# define ECerr(f,r)   ERR_raise(ERR_LIB_EC,(r))
-# define SSLerr(f,r)  ERR_raise(ERR_LIB_SSL,(r))
-# define BIOerr(f,r)  ERR_raise(ERR_LIB_BIO,(r))
-# define PKCS7err(f,r) ERR_raise(ERR_LIB_PKCS7,(r))
-# define X509V3err(f,r) ERR_raise(ERR_LIB_X509V3,(r))
-# define PKCS12err(f,r) ERR_raise(ERR_LIB_PKCS12,(r))
-# define RANDerr(f,r) ERR_raise(ERR_LIB_RAND,(r))
-# define DSOerr(f,r) ERR_raise(ERR_LIB_DSO,(r))
-# define ENGINEerr(f,r) ERR_raise(ERR_LIB_ENGINE,(r))
-# define OCSPerr(f,r) ERR_raise(ERR_LIB_OCSP,(r))
-# define UIerr(f,r) ERR_raise(ERR_LIB_UI,(r))
-# define COMPerr(f,r) ERR_raise(ERR_LIB_COMP,(r))
-# define ECDSAerr(f,r)  ERR_raise(ERR_LIB_ECDSA,(r))
-# define ECDHerr(f,r)  ERR_raise(ERR_LIB_ECDH,(r))
-# define OSSL_STOREerr(f,r) ERR_raise(ERR_LIB_OSSL_STORE,(r))
-# define FIPSerr(f,r) ERR_raise(ERR_LIB_FIPS,(r))
-# define CMSerr(f,r) ERR_raise(ERR_LIB_CMS,(r))
-# define CRMFerr(f,r) ERR_raise(ERR_LIB_CRMF,(r))
-# define CMPerr(f,r) ERR_raise(ERR_LIB_CMP,(r))
-# define TSerr(f,r) ERR_raise(ERR_LIB_TS,(r))
-# define HMACerr(f,r) ERR_raise(ERR_LIB_HMAC,(r))
-# define CTerr(f,r) ERR_raise(ERR_LIB_CT,(r))
-# define ASYNCerr(f,r) ERR_raise(ERR_LIB_ASYNC,(r))
-# define KDFerr(f,r) ERR_raise(ERR_LIB_KDF,(r))
-# define SM2err(f,r) ERR_raise(ERR_LIB_SM2,(r))
-# define ESSerr(f,r) ERR_raise(ERR_LIB_ESS,(r))
-# define PROPerr(f,r) ERR_raise(ERR_LIB_PROP,(r))
-# define PROVerr(f,r) ERR_raise(ERR_LIB_PROV,(r))
```

Comment:
You know, rearranging the order makes it a bit difficult to check if you got it right...  fortunately, Travis is here to chastise you, so see this as a gentle reminder to add `FUNCerr` back

Reply:
No, FUNCerr was added to master so it doesn't have to come back.
This PR is not intended to compile until all the existing old macros are converted.

Reply:
OK then